base/src/types.rs

```diff
@@ -351,6 +352,64 @@ pub enum FontScheme {
     None,
 }
 
+// MergedRange type
+// There will be one MergedRange struct maintained for every Merge cells that we load
+// merge_cell_range : Its tuple having [row_start, column_start, row_end, column_end]
+#[derive(Encode, Decode, Debug, PartialEq, Eq, Clone)]
+pub struct MergedRange(i32, i32, i32, i32);
+
+// implementing accessor function
+impl MergedRange {
+    // Accessor methods
+    pub fn get_row_start(&self) -> i32 {
+        self.0
+    }
+
+    pub fn get_col_start(&self) -> i32 {
+        self.1
+    }
+
+    pub fn get_row_end(&self) -> i32 {
+        self.2
+    }
+
+    pub fn get_col_end(&self) -> i32 {
+        self.3
+    }
+
+    // Method which returns range_ref from the tuple
+    // ex : (3,1,4,2) is interpreted as A3:B4
+    // This public method does not return the output wrapped in Result
+    // idea is, once MergedRange struct gets created, there shoud not be any problem in converting
+    // the tuple in to the range_ref string. so purposefully number_to_column fn is not handled
+    // if it panics here, then it will be a bug as in corner scenario which we had not thought while creating Merge cell
+    pub fn get_merge_range_as_str(&self) -> String {
+        let st_col = number_to_column(self.get_col_start()).expect(&format!(
```

Comment:
We can't use `expect`, that is similar to `unwrap` or `panic`. we can't panic in the library

Reply:
Also, please don't use `col`, use `column` (I know brain dead, but I use it all over)